test/tsickle_test.ts

```diff
@@ -21,6 +21,7 @@ describe('emitWithTsickle', () => {
       customTransformers?: tsickle.EmitTransformers): {[fileName: string]: string} {
     const tsCompilerOptions:
         ts.CompilerOptions = {...testSupport.compilerOptions, ...tsConfigOverride};
+    tsCompilerOptions.outDir = path.join(tsCompilerOptions.rootDir!, 'tmp');
```

Comment:
I'm confused again.  Does this mean, even without tsickle involved, you cannot use resolveJsonModule without setting an outDir because the compiler wants to overwrite the input files?

Reply:
Yes.

Reply:
I see, wow!

Can you name this 'out' instead of 'tmp', just in case it shows up in an error message somewhere?

Reply:
Sure, done.